magicctapipe/resources/config_general.yaml

```diff
@@ -0,0 +1,28 @@
+mc_tel_ids:
+    LST-1: 1
+    LST-2: 0
+    LST-3: 0
+    LST-4: 0
+    MAGIC-I: 2
+    MAGIC-II: 3
+
+directories:
+    workspace_dir : "/fefs/aswg/workspace/raniere/" 
+    scripts_dir   : "/fefs/aswg/workspace/raniere/"
+    target_name   : "CrabTeste"
+    MC_gammas     : "/fefs/aswg/data/mc/DL0/LSTProd2/TestDataset/sim_telarray"
+    MC_electrons  : "/fefs/aswg/data/mc/DL0/LSTProd2/TestDataset/Electrons/sim_telarray/" 
+    MC_helium     : "/fefs/aswg/data/mc/DL0/LSTProd2/TestDataset/Helium/sim_telarray/" 
+    MC_protons    : "/fefs/aswg/data/mc/DL0/LSTProd2/TrainingDataset/Protons/dec_2276/sim_telarray"
+    MC_gammadiff  : "/fefs/aswg/data/mc/DL0/LSTProd2/TrainingDataset/GammaDiffuse/dec_2276/sim_telarray/"
```

Comment:
I need to check how exactly you are using those later on in the code, but I do not like much to have such tags like MC_gammas, MC_gammadiff, MC_protons, etc.. What if somebody want to do a study with iron nuclei? Or if you want to train with ring wobble instead of diffuse gammas (or other way around if you need diffuse gammas for testing because you analyze an extended source. 
It is better if you specify the pipeline if you process data or MCs and where to find it. Actually I would also prefer that the config includes only the information how to process the data, and the actual paths are specified as command line arguments to the scripts. This makes it easier to massive process the data/MCs in a cluster, and the config file is more general.

Reply:
now reaching till the end of the code, I think those directories are not used at all now, and they will be needed only with the next PR

Reply:
Yes, they will be needed only in the semi-automatic scripts.  We removed them from the config file, but we will take into account your comment in the next PR :)